src/App.js

```diff
@@ -1,7 +1,25 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import { Footer } from "./components/footer/Footer";
+import { Navbar } from "./components/navbar/Navbar";
+import { Home } from "./pages/Home/Home";
```

Comment:
try giving the route as navbar/Navbar.js, as it makes more sense, try implementing in other routes as well